src/__internal__/focus-trap/focus-trap-utils.js

```diff
@@ -1,12 +1,40 @@
 const defaultFocusableSelectors =
   'button:not([disabled]), [href], input:not([type="hidden"]):not([disabled]), select:not([disabled]), textarea:not([disabled]), [tabindex]';
 
+// callback-based rather than Promise-based, as Jest fake timers unfortunately don't work nicely with Promises
+const waitForCss = (element, expectedStyles, maxTime, callback) => {
```

Comment:
This function isn't called anywhere else so do we need to pass in these as parameters (apart from `element`) rather than setting them in the function body?